mantidproject/mantid
# D11B nexus format

**Description of work.**
Adjusts the LoadILLSANS to accommodate the converged nexus format for the new D11.



**To test:**
Load the newly added test file.
Check the sample logs `L2` and `Detector 1.det_calc`.
They should be the same and about 8m.
Check the instrument viewer, pick around the center of the back detector, make sure that `r` and `z` are also around 8m.

If you are at ILL, try to check the same with few other numors from this cycle's internalUse in the range 326-412.



*There is no associated issue.*

*This does not require release notes since it is part of commissioning.*




---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there is GUI work does it follow the [GUI standards](http://developer.mantidproject.org/Standards/GUIStandards.html)?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
